example/seed-crds/10-crd-autoscaling.k8s.io_hvpas.yaml

```diff
@@ -924,6 +924,9 @@ spec:
                                     mode:
                                       description: Whether autoscaler is enabled for
                                         the container. The default is "Auto".
+                                      enum:
+                                      - Auto
+                                      - "Off"
```

Comment:
Does this need to be updated in https://github.com/gardener/gardener/blob/7a20ba3dbd88cb3ca804da23bd2dae61236ca9cb/charts/seed-bootstrap-crds/charts/hvpa/templates/hvpa.yaml as well?

Also, please `/hold cancel` when you think the PR is ready

Reply:
I updated the above file to be in sync with [example/seed-crds/10-crd-autoscaling.k8s.io_hvpas.yaml](https://github.com/gardener/gardener/pull/5716/files/eed5f2d724acc672803725b84030b75ea1b47dee#diff-698b02bf865a5a3d0a68bd125bf2fca4429b22ded546c436a840587d029cdaca) (as there were a few other minor differences as well.

I am still seeing one issue with 0.10.0 that was not present with 0.9.2, albeit a much less severe one. Also, there are a few issues with 0.10.0 that have been reported (see https://github.com/kubernetes/autoscaler/issues?q=is%3Aissue+label%3Avertical-pod-autoscaler+is%3Aopen+) and not yet properly analyzed or responded to. We are still not sure whether 0.10.0 is safe to use in production, so I will keep this PR on hold until it's more clear.

Reply:
Finally, after spending some more time testing I wasn't able to reproduce neither the issues that I initially saw, nor some of the other open issues with 0.10.0. It seems they were all consequences of https://github.com/kubernetes/autoscaler/issues/4733 and were meanwhile closed. So we'll proceed with merging this PR.